### PR TITLE
Create 2017-08-07-mergermotions.md

### PR DESCRIPTION
**Pull request by raynamharris**
Announcement about merger motions

### CODE REVIEW

**Comment by raynamharris** [comment]
Also posted here: https://github.com/datacarpentry/datacarpentry.github.io/pull/267

File: _posts/2017/08/2017-08-07-mergermotions.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+authors: ["Rayna Harris"]
+title: "Motions approved for Data Carpentry & Software Carpentry Merger"
+date: 2017-08-07
+time: "16:00"
+category: ["Steering Committee", "Community", "Data Carpentry"]
+---
+
+I am happy to announce that the Ssteering Committees of both Software Carpentry and Data Carpentry have approved 4 motions regarding the structure and leadership of the mergered Carpentries organization. The approved motions are:
```
Comment (k8hertweck) [inline _posts/2017/08/2017-08-07-mergermotions.md]: "Steering Committee" seems to be hissing here. Make us less snakey!

**Comment by k8hertweck** [comment]
One minor suggestion, thanks for adding this!

File: _posts/2017/08/2017-08-07-mergermotions.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+title: "Motions approved for Data Carpentry & Software Carpentry Merger"
+date: 2017-08-07
+time: "16:00"
+category: ["Steering Committee", "Community", "Data Carpentry"]
+---
+
+I am happy to announce that the Ssteering Committees of both Software Carpentry and Data Carpentry have approved 4 motions regarding the structure and leadership of the mergered Carpentries organization. The approved motions are: 
+
+## Motion 1
+The Board of Directors for the combined organization will be composed of 9 members, each serving a two year term without limits on the number of terms. Five members will be appointed through a process of nomination to the board followed by voting by board members. The other four members will be elected by the membership of the organization.
```
Comment (gvwilson) [inline _posts/2017/08/2017-08-07-mergermotions.md]: Nomination by whom?

File: _posts/2017/08/2017-08-07-mergermotions.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+**Background:**  The ED is the link between the Board and the operations of the organization. The ED will have autonomy to make decisions about running the organization, given strategic direction from the Board. 
+
+## Motion 3
+The combined organization will have a Director of Business Development who reports to the Executive Director. Initially, this position will be offered to Jonah Duckles. 
+
+**Background:** Business development is critical to the long-term sustainability of the organization. In the merger of two organizations, each with an ED, this clarifies roles and reporting.
+
+## Motion 4
+Existing subcommittees and task forces will have a point of contact from among the staff, rather than reporting directly to the Board of Directors.
```
Comment (gvwilson) [inline _posts/2017/08/2017-08-07-mergermotions.md]: Wants a blank link before `**Background**` for consistency with other points.

**Review by gvwilson — COMMENTED** [review]
Great news!

**Comment by tracykteal** [comment]
Thanks @raynamharris! We still can't use the term 'board'. I know these were in the motions, but can they be changed for the blog post?

**Pull request merged**